Sources/LambdaKit/UIImagePickerController+LambdaKit.swift

```diff
@@ -78,8 +78,9 @@ extension UIImagePickerController: UIImagePickerControllerDelegate, UINavigation
 
     // MARK: UIImagePickerControllerDelegate implementation
 
-    open func imagePickerController(_ picker: UIImagePickerController,
-        didFinishPickingMediaWithInfo info: [String : Any])
+    open func imagePickerController(
+        _ picker: UIImagePickerController,
+        didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any])
```

Comment:
`.InfoKey: Any`

Reply:
That can't be inferred

Reply:
Sorry, just meant the spacing, not the type name

Reply:
ah yea, migrator

Reply:
oh actually it was wrong before